front-end/src/views/function/components/AddFunc.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+  <el-drawer
+    ref="addFuncDrawer"
+    size="40%"
+    title="AddFunction"
+  >
+    <el-form ref="addForm"
+             class="addFunc"
+             :rules="rules"
+             :model="form"
+             label-width="80px"
+             label-position="top"
+             style="width: 90%;margin: 25px"
```

Comment:
Codacy has a fix for the issue: [Replace `·············style="width:·90%;` with `······style="width:·90%;·`](https://app.codacy.com/gh/FunctionStream/function-stream/pullRequest?prid=7834759)
```suggestion
      style="width: 90%; margin: 25px"
```